docs/source/maintenance/migration-rocksdb.rst

```diff
@@ -0,0 +1,151 @@
+====================
+Migration To RocksDB
+====================
+
+Iroha allows for using Postgres or Rocks Database.
+You can use the option you prefer and if you want to switch -- there is an option for you.
+**Just migrate your database!**
+Here is how
+
+.. hint:: Both migration tool and WSV checker have ``-help`` that you can call to check the available flags anytime.
```

Comment:
```suggestion
.. hint:: Both tools ``iroha_migrate`` and ``iroha_wsv_diff`` have ``-help`` that you can call to check the available flags anytime.
```

Reply:
I wanted to use some sort of synonyms, so it is not only tech stuff and "" but also some nice to read English. I could call the checker "tool that checks the differences in WSV"? The more synonyms there are, the more understanding of the concept.